parm/config/gefs/config.efcs

```diff
@@ -38,26 +38,22 @@ if [[ "${RUN}" == "enkfgfs" ]]; then
    export FHMAX=${FHMAX_ENKF_GFS:-${FHMAX}}
 fi
 
-# Use serial I/O for ensemble (lustre?)
-export OUTPUT_FILETYPE_ATM="netcdf"
-export OUTPUT_FILETYPE_SFC="netcdf"
-
 # Stochastic physics parameters (only for ensemble forecasts)
 export DO_SKEB="YES"
-export SKEB=0.3
-export SKEB_TAU=21600.
-export SKEB_LSCALE=250000.
-export SKEBNORM=0
+export SKEB="0.8,-999,-999,-999,-999"
+export SKEB_TAU="2.16E4,1.728E5,2.592E6,7.776E6,3.1536E7"
```

Comment:
though tau for second scale is not used at this time, but we need to correct that value.
here is the correct one. 
SKEB_TAU="2.16E4,2.592E5,2.592E6,7.776E6,3.1536E7"

Reply:
code updated